src/components/DocumentsView/DocumentView.tsx

```diff
@@ -27,10 +43,15 @@ export const DocumentView: FunctionComponent<IDocumentViewProps> = (props) => {
         <Card className="sbo-gray-gradiant">
             <Row className="m-3">
                 <Col>
-                    <a download href={props.document.LinkUrl}>
+                    <a download={!isOfficeFile} href={getDownloadUrl()}>
                         <Icon {...getFileTypeIconProps({ extension: extension, size: 20, imageFileType: 'png' })} className="show-overflow" />
```

Comment:
IE is cutting off the bottom of the icons. Nothing obvious as to why. Seems like an IE oddity. If you want to take a quick look, feel free. Otherwise I'm not too worried about it.

Reply:
I noticed this, I didn't really want to try to fix it unless it was pointed out lol

Reply:
Fixed with some CSS